src/components/common/ExerciseCard.jsx

```
@@ -0,0 +1,257 @@
+                </Button>
+              </div>
+            )}
+          </div>
+
+          {/* Exercise controls - responsive grid */}
+          <div className="grid grid-cols-3 gap-2 sm:gap-3 lg:gap-4 mb-3 sm:mb-4 lg:mb-6">
+            {/* Sets */}
+            <div className="space-y-1 sm:space-y-2">
+              <label className="block text-xs sm:text-sm font-medium text-gray-300 text-center">
+                Sets
+              </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)